src/color-picker/color-picker.js

```diff
@@ -84,9 +83,13 @@ const Self = class ColorPicker extends NudeElement {
 				slider.color = this.color;
 			}
 
-			if (this.color && (!this._el.swatch.color || !this.color.equals(this._el.swatch.color))) {
+			if (!this._el.swatch.color || !this.color.equals(this._el.swatch.color)) {
 				// Avoid typing e.g. "red" and having it replaced with "rgb(100% 0% 0%)" under your caret
-				prop.applyChange(this._el.swatch, { ...change, source: "property" });
+				if (change.source === "attribute") {
+					// <color-swatch> doesn't have a color attribute, so we must change the corresponding property
```

Comment:
it …does? https://elements.colorjs.io/src/color-swatch/#the-color-attribute

Reply:
Actually, this is not the case after [we fixed `<color-swatch>`](https://github.com/color-js/elements/issues/85) (by replacing `defaultProp` with a getter and setter). I'll remove this check with the comment entirety. Removing the code and making our components even lighter is so cool. 🤩